.github/workflows/cherry-pick-to-release-branch.yml

```diff
@@ -0,0 +1,19 @@
+name: PR for release branch
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release_pull_request:
+    if: "contains(github.event.head_commit.message, 'fix')"
+    runs-on: ubuntu-latest
+    name: release_pull_request
+    steps:
+      - name: checkout
+        uses: actions/checkout@v1
+      - name: Create PR to branch
+        uses: risingwavelabs/github-action-cherry-pick@master
+        with:
+          pr_branch: 'v0.19.0-rc'
```

Comment:
Recommend adding a label like `cherry-pick` to identify it's a cherry-pick PR and adding PR description like below to reference the original PR.
```
labels: |
            cherry-pick
body: 'Cherry picking #{old_pull_request_id} onto this branch'
```